tests/Number/RealTest.php

```diff
@@ -63,12 +63,35 @@ public function testIfRealIsIntegerOrNotShouldSuccess()
         $this->assertTrue($i->isInteger());
     }
 
-
     public function testStringContextShouldSuccess()
     {
         $i = new Real(2.56);
         $d = new Real(0.56);
         $this->assertEquals('2.56', "$i");
         $this->assertEquals('0.56', "$d");
     }
+
+    public function testGetShouldBePI()
+    {
+        $real = new Real(2.56);
+        $this->assertEquals(M_PI, $real->__get('pi')->__toString());
```

Comment:
Hum… I dont know, this is strange to get pi, e, and so on from an instance. Confusing.

But, we can do that: 
```php
$real = new Real(Real::PI);
$this->assertEquals(M_PI, "$real");
```